bentoml/_internal/models/model.py

```diff
@@ -490,12 +488,19 @@ def __init__(
         api_version: str | None = None,
         creation_time: datetime | None = None,
     ):
-        ...
-
-    def __attrs_post_init__(self):
-        object.__setattr__(self, "name", self.tag.name)
-        object.__setattr__(self, "version", self.tag.version)
-
```

Comment:
should we use `field_transformer` instead here?

Reply:
The current approach is ok

Reply:
the "__attrs_init__" is the recommended way in case the initialization parameter type of the class is not the same as the property type. Naturally friendly to typing tools.